docs/kyma/04-10-upgrade.md

```diff
@@ -15,7 +15,7 @@ For example, if you're running Kyma 1.0 and you want to upgrade to version 1.3,
 2. Upgrade from version 1.1 to version 1.2.
 3. Upgrade from version 1.2 to version 1.3.
 
->**NOTE:** Kyma does not support a dedicated downgrade procedure. You can achieve a similar result by restoring your cluster from a backup. Read [this](/components/backup/#overview-overview) document to learn more about backups.
+>**NOTE:** Kyma does not support a dedicated downgrade procedure. You can achieve a similar result by taking a backup from your cluster before upgrade. Read [this](/components/backup/#velero-tutorial) document to learn more about backups.
```

Comment:
Please change link to tutorial to `/components/backup/#tutorials-taking-backup-using-velero`. You can see how links will look in website from preview of your PR -> https://deploy-preview-8261--kyma-project-docs-preview.netlify.app/components/backup/#tutorials-taking-backup-using-velero We have a concatenation of group of docs (in this case `tutorials`) and title of md :)

Reply:
Done, thanks :)